templates/space/show.html.twig

```diff
@@ -5,25 +5,27 @@
 
 {% block body %}
 <div class="carousel">
-            <div class="carousel-item-mc carousel-item-visible">
-                <img src="{{ asset('uploads/' ~ space.photo) }}" alt="bureau privee" />
-            </div>
-            <div class="carousel-item-mc">
-                <img src="{{ asset('build/images/salledereunion.jpg') }}"
-                    alt="salle de reunion" />
-            </div>
-            <div class="carousel-item-mc">
-                <img src="{{ asset('build/images/openspace.jpg') }}"
-                    alt="open space" />
-            </div>
-            <div class="carousel-item-mc">
-                <img src="{{ asset('build/images/espacedestockage.jpg') }}"
-                    alt="espace de stockage" />
-            </div>
-            <div class="carousel-actions">
-                <button id="carousel-button-prev" aria-label="Previous"><</button>
-                <button id="carousel-button-next" aria-label="Next">></button>
-            </div>
+	{% if space.images[0] is defined %}
```

Comment:
this:

    {% if space.images[0] is defined %}

should be:

    {% if space.images|length > 0 %}